File: eBordo.Model/Models/Stadion.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eBordo.Model.Models
{
    public class Stadion
    {
        public int stadionId { get; set; }
        public string nazivStadiona { get; set; }
        public Grad lokacijaStadiona { get; set; }
        public int lokacijaStadionaId { get; set; }
        public byte[] slikaStadiona { get; set; }
    }
}

File: ebordo_mobile/lib/models/stadion.dart
import 'dart:convert';
import 'package:ebordo_mobile/models/grad.dart';

class Stadion {
  final int stadionId;
  final String nazivStadiona;
  final Grad lokacijaStadiona;
  final List<int> slikaStadiona;

  Stadion({
    required this.stadionId,
    required this.nazivStadiona,
    required this.lokacijaStadiona,
    required this.slikaStadiona,
  });

  factory Stadion.fromJson(Map<String, dynamic> json) {
    String stringByte = json["slikaStadiona"] as String;
    List<int> bytes = base64.decode(stringByte);
    return Stadion(
      stadionId: int.parse(json["stadionId"].toString()),
      nazivStadiona: json["nazivStadiona"],
      lokacijaStadiona: Grad.fromJson(json['lokacijaStadiona']),
      slikaStadiona: bytes,
    );
  }

  Map<String, dynamic> toJson() => {
        "stadionId": stadionId,
        "nazivStadiona": nazivStadiona,
        "lokacijaStadiona": lokacijaStadiona,
        "slikaStadiona": slikaStadiona
      };
}
